docs/contents/dev-docs/release.md

```diff
@@ -72,49 +72,45 @@ for SERVICE in sns s3; do
 done
 ```
 
-3. You can review the release artifacts that were built for each service by
-   looking in the `services/$SERVICE/helm` directory:
+3) You can review the release artifacts that were built for each service by looking in the `services/$SERVICE/helm`
+directory:
 
-    `tree services/$SERVICE/helm`
+`tree services/$SERVICE/helm`
 
-    or by doing:
+or by doing:
 
-    `git diff`
+`git diff`
 
 !!! note
-    When you run `scripts/build-controller-release.sh` for a service, it will
-    overwrite any Helm chart files that had previously been generated in the
-    `services/$SERVICE/helm` directory with files that refer to the
-    Docker image with an image tag referring to the release you've just built
-    artifacts for.
-
-4. Commit your code and create a pull request:
-
+    When you run `scripts/build-controller-release.sh` for a service, it will overwrite any Helm chart files that had
+    previously been generated in the `services/$SERVICE/helm` directory with files that refer to the Docker image with
+    an image tag referring to the release you've just built artifacts for.
+   
+4) Commit your code and create a pull request:
 ```bash
 git commit -a -m "release artifacts for release $RELEASE_VERSION"
 ```
 
-5. Get your pull request reviewed and merged.
-
-6. Upon merging the pull request
+5) Get your pull request reviewed and merged.
 
+6) Upon merging the pull request
 ```bash
 git tag -a $RELEASE_VERSION $( git rev-parse HEAD )
 git push upstream main --tags
 ```
 
-!!! todo
-    A Github Action should execute the above
-
-which will end up associating a Git tag (and therefore a Github release) with
-the SHA1 commit ID of the source code for the controllers and the release
-artifacts you built for that release version.
+!!! note "TODO"
+    A Github Action should execute the above step which will end up associating a Git tag (and therefore a Github
+    release) with the SHA1 commit ID of the source code for the controllers and the release artifacts you built for
+    that release version.
```

Comment:
How will you pass a `$RELEASE_VERSION` variable to the Github action? i'm afraid that this action will always need to be manually executed.

Reply:
The only non manual way i could think of is to extract the `RELEASE_VERSION` from the last commit message

Reply:
This TODO was there before the PR, so I just left it. But I understand your concern here.